shared/rakefiles/entrypoint.rake

```diff
@@ -21,6 +21,11 @@ end
 
 @serviceaccount_key_file = "secrets/kube-system/owner.json"
 
+desc "Pull the current exekube container from the Docker hub"
+task :update_exekube => :set_vars do
```

Comment:
Do you think we should update gitlab-ci.yml to call this instead of the bare `docker pull` command there?

Reply:
+1

Reply:
I think so. I added this task because it uses the same version found in the docker-compose.yml file. We can have the version of the exekube image in only one place.